plugins/modules/notification/pushover.py

```diff
@@ -38,8 +39,14 @@
     description:
       - Message priority (see U(https://pushover.net) for details).
     required: false
+  device:
+    description:
+      - A device the message should be sent to. Multiple devices can be specified, separated by a comma.
+    required: false
```

Comment:
```suggestion
    required: false
    version_added: 1.2.0
```

Reply:
Thanks for your review, @felixfontein, much appreciated. Do I have to specify somewhere else that this is version 1.2.0?

Reply:
No, this is fine. The changelog fragment will appear under 1.2.0 (the next release) when this is merged in time; the `version_added` is shown in the docs so that users know that they need version 1.2.0 or newer of this collection to get this feature.